commit-message-guidelines.md

```diff
@@ -0,0 +1,32 @@
+# Commit Message Guidelines
+
+[Atomic commits](https://www.aleksandrhovhannisyan.com/blog/atomic-git-commits/) are preferred.
+
+## Subject/Title
+
+* Separate the title/subject of the commit message from the body of the commit message with a blank line.
+* Try to keep the title/subject descriptive and concise.
+* Capitalize the title/subject line, use the imperative mood, and don't end the line with a period.
```

Comment:
I'd prefer the indicative mood, for example:

Fixes bug: isMobileVR variable shadowed by bespoke isMobileVR() function

or

Forces the repository name to be lowercase in the custom docker workflow